docs/guidance/build/data_collection_tools.md

```diff
@@ -0,0 +1,104 @@
+# Data collection tools
+
+Where contracting data is managed using IT systems, implementing OCDS involves identifying how to extract, convert and publish that data.
+
+Where contracting data is managed on paper or using unstructured electronic documents, implementing OCDS typically involves creating a bespoke IT system to collect data in a structured form.
+
+If you need to collect structured data, but you don’t have the capacity to create an IT system, you can consider reusing an existing data collection tool. These tools offer fewer opportunities for customization than a bespoke IT system, but can provide a quick route to collecting and publishing data in OCDS format.
+
+## Data collection form
+
+The [data collection form](https://www.open-contracting.org/resources/ocds-data-collection-form/) is a web-based form for collecting OCDS data.
+
+Data from the form is copied to a Google Sheet which structures and formats it to conform to OCDS.
+
+Data entered using the form can be checked and converted using the [OCDS Data Review Tool](https://standard.open-contracting.org/review/) and published in either spreadsheet or JSON format.
+
+Consider using the form if:
+
+* You don’t have the capacity to develop or install a software tool
+* Your users have access to reliable internet connections
+* Your users are unfamiliar with spreadsheets
+* Data entry will be done by many different users
+* You want to minimize the work required to collate data
+* You don’t need to update the data about a contracting process after it is first entered
+
+The form includes a subset of fields from the **tender** and **buyer** sections of OCDS. The OCDS Helpdesk can help you extend and adapt the form to suit your needs.
+
+Read more about using and customizing the form in the [resource guide](https://www.open-contracting.org/resources/ocds-data-collection-form/). The OCDS helpdesk can also provide guidance on using the form to collect data and help you to analyze the data that you collect.
+
+## Data collection spreadsheet
+
+The [data collection spreadsheet](https://www.open-contracting.org/resources/data-collection-spreadsheet/) is available in Google Sheets and for offline use in Microsoft Excel.
+
+Data is entered directly into the template, which structures and formats it to conform to OCDS.
+
+Data entered the template can be checked and converted using the [OCDS Data Review Tool](https://standard.open-contracting.org/review/) and published in either spreadsheet or JSON format.
+
+Consider using the spreadsheet template to collect data if:
+
+* You don’t have the capacity to develop or install a software tool
+* You need to collect data without access to an internet connection
+* Your users are familiar with using spreadsheets
+* Data entry will be done by a small number of users
+* You have the capacity to collate data entered in multiple spreadsheets
+* You need to make multiple updates over the life of a contracting process
```

Comment:
Consider adding **emphasis** around some key words that distinguish each list from the other lists, e.g. **without** access, are **familiar**, by a **small number of**, etc.

Reply:
Done in https://github.com/open-contracting/standard/pull/1061/commits/9aa990a192ed4e8bc1135bd7e12c24fb0b1693b0, feel free to suggest changes